usaspending_api/common/tests/integration/test_decorators.py

```diff
@@ -0,0 +1,60 @@
+# Stdlib imports
```

Comment:
Can a test be added for using the default timeout in settings? 

To avoid a 5min test, you'd have to feed the test runner different settings. Can you override settings for a single test?

Reply:
You can, i can add a test that mocks the settings value to test the default